src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -37,6 +38,31 @@ namespace Microsoft.PowerFx.Core.Functions
     [ThreadSafeImmutable]
     internal abstract class TexlFunction : IFunction
     {
+        /// <summary>
+        /// Function can override this method to provide BlankHandling policy for argument.
+        /// </summary>
+        /// <param name="index">0 based Index of argument.</param>
+        /// <returns></returns>
+        public virtual BlankHandlerPolicy GetArgBlankHandlerPolicy(int index)
```

Comment:
>BlankHandlerPolicy [](http://example.com/codeflow?start=23&length=18)

nit - can we move this to the bottom of the file?

Reply:
moved